interface/patient_file/encounter/superbill_custom_full.php

```diff
@@ -260,12 +269,41 @@ function bucks($amount)
 <head>
     <title><?php echo xlt("Codes"); ?></title>
     <?php html_header_show(); ?>
+
     <link rel="stylesheet" href="<?php echo attr($css_header);?>" type="text/css">
     <script type="text/javascript" src="../../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
     <script type="text/javascript" src="../../../library/textformat.js"></script>
-
-    <script language="JavaScript">
-
+    <?php if ($institutional) { ?>
+    <script type="text/JavaScript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-3-1-1/index.js"></script>
+    <link href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-ui-1-12-1/themes/base/jquery-ui.min.css" rel="stylesheet" type="text/css" />
+    <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-ui-1-12-1/jquery-ui.min.js"></script>
```

Comment:
to aovid confusion down the road, rec placing the above 3 asset with the other assets prior to the if $institutional